src/manifest/dbtWorkspaceFolder.ts

```diff
@@ -102,13 +98,42 @@ export class DBTWorkspaceFolder implements Disposable {
       {},
       { numProjects: projectFiles.length },
     );
-    return projectFiles.forEach((uri) => this.registerDBTProject(uri));
+    await Promise.all(
+      projectFiles.map(async (uri) => {
+        await this.registerDBTProject(uri);
+      }),
+    );
+    // Filter projects
+    let packagesInstallPaths = await this.findDBTProjectPackagesInstallPaths();
+
+    packagesInstallPaths = packagesInstallPaths.filter(
```

Comment:
should this filter to be inside `findDBTProjectPackagesInstallPaths`?

Reply:
Fixed